axum-macros/tests/debug_handler/fail/wrong_return_tuple.rs

```diff
@@ -0,0 +1,11 @@
+
+#[axum::debug_handler]
+async fn handler() -> (
+    axum::http::StatusCode,
+    axum::Json<&'static str>,
+    axum::response::AppendHeaders<[( axum::http::HeaderName,&'static str); 1]>,
+) {
```

Comment:
I think it would be nice to have a third test like this but with a type that isn't special cased. So a type made by the user that implements `IntoResponse` and isn't the last item. To see that the error is also good there.